src/components/layout/AppLayout.tsx

```diff
@@ -10,6 +11,10 @@ type AppLayoutProps = {
 }
 
 export const AppLayout: React.FC<AppLayoutProps> = ({ children, hideSideNav, sx }) => {
+  const theme = useTheme()
+  const matchMobile = useMediaQuery(theme.breakpoints.down('sm'))
+  const interopRoutes = useGetSidebarItems()
```

Comment:
Let's avoid having a logic based on jwt in this part of the `AppLayout` which is rendered also on public pages